handbook/engineering/observability/monitoring_pillars.md

```diff
@@ -63,28 +35,61 @@ Before the constraints we impose will start to make sense, you need to understan
 
 You can think of the restrictions we impose as a linter for preventing the issues described above. And, in many cases, our tooling does already prevent these issues by imposing these restrictions!
 
-## Frequently asked questions
+## The Sourcegraph monitoring pillars
+
+Monitoring tooling at Sourcegraph is developed to encourage the following guidelines:
+
+- [**Dashboards should only be created with the monitoring generator**](#dashboards-should-only-be-created-with-the-monitoring-generator)
+- [**Dashboards should describe a single service**](#dashboards-should-describe-a-single-service)
+- [**Graphs should have associated alerts**](#graphs-should-have-associated-alerts)
+- [**Graphs should have less than 5 cardinality**](#graphs-should-have-less-than-5-cardinality)
+- [**Less useful graphs should be hidden by default**](#less-useful-graphs-should-be-hidden-by-default)
+
+Before trying to circumvent the guidelines enforced by our monitoring tooling, please keep in mind that these pillars are defined to help us achieve our [goals](#long-term-vision) and alleviate [pain points we have encountered in the past](#history). Each of the above pillars are documented with detailed explanations, recommendations for what you can do, and exceptions where applicable.
+
+To make significant changes to the monitoring tooling, please reach out to the [Distribution team](../distribution/index.md) or open a PR to this page!
+
+### Dashboards should only be created with the monitoring generator
+
+Creating dashboards outside of the monitoring generator, such as with the Grafana WYSIWYG editor, is discouraged. All metrics should be presented in the most plain, mundane, non-fanciful way that every site admin can understand.
+
+#### Explanation
 
-### FAQ: Why can't I create an ad-hoc dashboard (e.g. an "HTTP" dashboard)?
+Using the Grafana WYSIWYG editor, or otherwise operating outside the constraints of [the monitoring generator](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/monitoring) makes it:
+
+- Extremely easy to introduce dashboards that do not comply with the constraints we want to impose.
+- Extremely easy to introduce dashboards that have inconsistency errors through mistakes that are easy to make (e.g. not using UTC time on dashboards, the preset timeframe being inconsistent, it not self-describing to admins what it does, etc.)
+- Difficult for others to modify and review without introducing regressions (e.g. due to Grafana schema version changes, which happen frequently, producing large diffs)
+- Much harder to declare proper alerts in our regular high-level alerting architecture.
+
+To learn more about what you get for free by using the monitoring generator, see [TODO](#TODO).
```

Comment:
I plan on figuring out a place to write up the perks of the generator somewhere! However I'm currently thinking of doing it _not_ in the handbook but directly in the `sourcegraph/sourcegraph` repo, under `monitoring` (if anyone thinks that's a bad idea let me know!)

Reply:
SGTM